### PR TITLE
Payf 3534 updates

### PR DESCRIPTION
**Pull request by jonhageman**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1377,22 +1377,22 @@ <h4 id="example-get-all-patient-types">Example: Get Payment Profiles By Customer
...
-        profileName : profileName,
+        profileName : 'profileName',
         isDefault : true,
-        cardNumberLast4 : cardNumberLast4,
-        customerId : customerId,
-        id : 0,
-        expiry : expiry,
-        creditCardType : creditCardType     
+        cardNumberLast4 : 'cardNumberLast4',
+        customerId : 'customerId',
```
Comment (l-rodriguez) [inline index.html:1383]: 'customerId' should be the same as sent on URL (line 1370 : 
"GET https://payfac.nextech.com/api/public/paymentProfiles/1738")

File: index.html [hunk 2]
```diff
@@ -1442,9 +1442,9 @@ <h4 id="example-get-all-patient-types">Example: Create Payment Profile Response<
     profileName : 'profileName',
     isDefault : true,
     cardNumberLast4 : 'cardNumberLast4',
-    customerId : 'customerId,
-    id : 0,
-    expiry : 'expiry,
+    customerId : 'customerId',
```
Comment (l-rodriguez) [inline index.html:1445]: remove "/15" from URL => "POST https://payfac.nextech.com/api/public/paymentProfiles/15" line (1426)

File: index.html (hunk 1 above)
Comment (jonhageman) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (jonhageman) [reply index.html]: done

**Review by l-rodriguez — APPROVED** [review]
lgtm!

**Pull request merged**